### DISCUSSION TITLE
What does "update" mean in the CLI?

### DISCUSSION THREAD

**Discussion by YouCanNotBeSerious** [HomeSpan/HomeSpan#760] [Q&A] [answered]
13:52:48.233 -> Service                             UUID         AID  IID  Update  Loop  Button  Linked Services
13:52:48.233 -> ------------------------------  --------  ----------  ---  ------  ----  ------  ---------------
13:52:48.265 -> AccessoryInformation                  3E           1    1      NO    NO      NO  -
13:52:48.265 -> AccessoryInformation                  3E           2    1      NO    NO      NO  -
13:52:48.265 -> TemperatureSensor                     8A           2    4      NO   YES      NO  -
13:52:48.265 -> 

My remote temperature sensor is not responding (status in the Home/Eve app) and I notice "update" is "no" above.

I took working spanpoint contact sensor code and changed services etc based on the temp sensor example... but the temp value never updates from the hardcoded default in setup() and the accessory is "not responding" in HomeKit apps.  I have homed in on the "update" column above, what does it mean?

**Comment by HomeSpan** [comment, marked as answer]
"Update" means that you added an update() function to the Service.  This would not be needed in a read-only Service such as a sensor.

**Reply by YouCanNotBeSerious** [reply]
Thanks, I was on the wrong track - overnight as a result of unplugging and rebooting both ends of the homespan temperature sensors it all started working (pairing not removed, code not updated) so I must have had something cached either at the iOS end or my code.
